### DISCUSSION TITLE
Load plugin before which-key?

### DISCUSSION THREAD

**Discussion by hotgazpacho** [LazyVim/LazyVim#472] [Q&A] [answered]
I'd like to try out [legendary.nvim](https://github.com/mrjones2014/legendary.nvim), which adds a command palette like VSCode. It can automatically discover the `which-key` configuration tables, but in order to do so, it needs to be loaded before `which-key`. I'm aware of the `priority` key in the plugin specs, but I'm not sure what this should be set to in order to have it load before `which-key`, or if that will even work.

Any thoughts?

Thanks!

**Comment by folke** [comment, marked as answer]
just add legendary as a dep to which-key, then it will always be loaded before which-key

**Reply by hotgazpacho** [reply]
Perfect, thanks!
